elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -0,0 +1,66 @@
+{{#question}}
+    <div class="row">
+        <div class="column {{dropzone_layout}}">
+            <ul id="{{answer_name}}-options" name="{{answer_name}}" class="connectedSortable">
+                <li class='ui-sortable-handle info info-fixed'>{{header-left-column}}</li>
+                    {{#options}}
+                        <li string="{{.}}" title="{{.}}">{{.}}</li>
+                    {{/options}}
+                </ul>
+        </div>
+            <div class="column {{dropzone_layout}}">
+                <ul id="{{answer_name}}-dropzone" name="{{answer_name}}" class="connectedSortable dropzone">
+                    <li class="ui-sortable-handle info info-fixed">{{header-right-column}}</li>
+                    {{#submission_dict}}
+                        <li string="{{text}}" title="{{text}}" style='margin-left: {{indent}}px;'>{{text}}</li>
+                    {{/submission_dict}}
+                </ul>
+            </div>
+        </div>
+    <input id="{{answer_name}}-input" type="hidden" name="{{answer_name}}-input" value=""/>
+
+{{/question}}
+
+{{#submission}}
+    {{^parse-error}}
+        <div style="margin-bottom: 5px;"><strong>Your answer: </strong>
+        {{#student_submission}}
+        	{{#render_as_code}}
+        		<p class="answer_block"> {{text}} </p> 
+        	{{/render_as_code}}
+        	{{^render_as_code}}
+        		<p> {{text}}</p>
+        	{{/render_as_code}}
+        {{/student_submission}}
+        {{#perfect_score}}
+            <span class="badge {{colour}}"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>
+        {{/perfect_score}}
+        {{^perfect_score}}
+                <span class="badge {{colour}}"><i class="fa fa-times" aria-hidden="true"></i>
+        {{score}}%</span> {{feedback}}
+        {{/perfect_score}}
+        </div>
+    {{/parse-error}}
+
+    {{#parse-error}}
+        <div style="margin-bottom: 5px;"><strong>Your answer: </strong></div>
+        <div id="pl-dropdown-{{uuid}}" class="d-inline-block"> 
+            <span class="badge text-danger badge-invalid"><i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Invalid</span>
+            <a href="javascript:void(0);" role="button" class="btn btn-sm btn-secondary small border" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" style="margin-left: 5px" title="Format Error" tabindex="0" data-content="{{parse-error}}"> Why <i class="fa fa-question-circle" aria-hidden="true"></i></a>
```

Comment:
Why do we have `href="javascript:void(0);"` here? This shouldn't be necessary for a bootstrap popover.

Reply:
I think this was from the mustache files from the other elements that I copied. I don't think removing the line breaks anything, so I will be removing it as you suggested.